src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -0,0 +1,103 @@
+---
+
+# Summary of how GitHub Issues and Pull Requests are processed
+
+GitHub is the main tool used by maintainers to manage the PrestaShop project.
+
+This page quickly describes how [Issues](https://guides.github.com/features/issues/) and [Pull Requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) are processed on GitHub.
+
+## The teams
+
+On GitHub, there are mainly three teams working together on the project.
+
+- Product Team: this team takes care of the product vision of the software. It has Product Managers, UX Designers and Wording Managers.
+- Quality Assurance (QA) team: this team takes care of ensuring the quality of the software. It has QA analysts and QA automation engineers.
+- Developers team a.k.a Core Team: this team takes care of the code and act as maintainer for the project.
+
+A maintainer will naturally belong to the Developers team but can also assist and work with the Product Team and QA Team.
+
+## Issues
+
+Currently, there are 4 different types of Issues submitted by users on GitHub.
+
+### Feature Request
+
+When user submits a Feature Request, it can be either a Product feature request (example: add a new Back office page) or a Technical feature request (example: support PostgreSQL).
+
+Product feature requests are labelled "Waiting for PM" while Technical feature request are labelled "Waiting for dev".
+
+The team in charge will then analyse the request and either reject it or accept it. Rejected feature requests are closed while accepted feature requests are added to the roadmap and labelled. Labels will help triaging the backlog.
+
+### Bug report
+
+When user submits a bug report, the QA team will analyse it and attempt to reproduce it. If it can be reproduced and is confirmed to be an issue, it will be labelled and added to the roadmap.
+
+If it cannot be reproduced, QA team will attempt to explore the issue with the contributor to isolate the very settings responsible for the buggy behavior.
+
+If user does not answer for 30 days or after multiple attempts, it cannot be reproduced on our side, the issue is closed.
+
+#### Regressions
+
+If the bug report is confirmed, one of the key elements to evaluate is whether it is a regression. A regression is a bug that cannot be observed in the previous PrestaShop version, it means the software quality level has decreased instead of increasing. Regressions are usually milestoned to be fixed in the next patch version.
+
+#### This is not easy
+
+The work of analyzing and testing all submitted bug reports is a very complex one, because there might be a very diverse range of reports. Moreover quite a huge number of them are actually not related to the software but to how the shop is being used: the server configuration, the shop configuration, the installed modules and the installed theme might introduce buggy behaviors that user mistakenly believe come from the software.
```

Comment:
```suggestion
The work of analyzing and testing all submitted bug reports is a very complex one, because there might be a very diverse range of reports. Moreover quite a huge number of them are actually not related to the software but to how the shop is being used: the server configuration, the shop configuration, the installed modules and the installed theme might introduce buggy behaviors that the user mistakenly believes come from the software.
```